civis/resources/_resources.py

```diff
@@ -467,9 +469,17 @@ def parse_api_spec(api_spec, api_version, resources):
     classes = {}
     for path, ops in paths.items():
         base_path, methods = parse_path(path, ops, api_version, resources)
-        class_name = to_camelcase(base_path)
+        class_name = base_path.title()
```

Comment:
This is the main issue that this PR addresses. The class name would look like "Foo_Bar" instead of "FooBar" after this PR. This change doesn't seem to be Pythonic for what class names are supposed to look like, but it really has no effect to the users, but no-one is really expected to access or use these generated classes in any way.

Before this PR, the details got swallowed by the "alias of" bit:
<img width="402" alt="Screen Shot 2021-12-03 at 9 10 50 AM" src="https://user-images.githubusercontent.com/86482098/144617575-7dd1f93f-6b73-483e-9ec8-c97c85606f3a.png">

After this PR (also added a code snippet for all endpoints):
<img width="722" alt="Screen Shot 2021-12-02 at 9 12 23 PM" src="https://user-images.githubusercontent.com/86482098/144617684-8123ddd0-0a29-4fe2-b648-c25af4fd7e28.png">